heron/cli/src/python/submit.py

```diff
@@ -89,7 +89,7 @@ def launch_a_topology(cl_args, tmp_dir, topology_file, topology_defn_file):
       "--release_file", release_yaml_file,
       "--topology_package", topology_pkg_path,
       "--topology_defn", topology_defn_file,
-      "--topology_jar", topology_file
+      "--topology_jar", topology_file   # pex file if pex specified
```

Comment:
Should be change it to --topology_binary and change it in the Java file? We can do it the next PR.